### DISCUSSION TITLE
Sensor not seeing traffic, even though bond0 is

### DISCUSSION THREAD

**Discussion by Mich-b** [Security-Onion-Solutions/securityonion#11106] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have the same issue as in https://github.com/Security-Onion-Solutions/securityonion/discussions/6647
I thought I'd simply wait for a couple of days for traffic to show up, but it does not show up. I have two NICs: one for management one which receives the mirrored traffic. Note that I'm using Proxmox and VMBR interfaces, this exact same setup worked in 2.3. 

Running `sudo tcpdump -e -v -n -i bond0` shows lots of traffic. 
ip a:

```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: ens18: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether 26:b2:70:e5:23:c8 brd ff:ff:ff:ff:ff:ff
    altname enp0s18
    inet 192.168.2.4/24 brd 192.168.2.255 scope global noprefixroute ens18
       valid_lft forever preferred_lft forever
3: ens19: <BROADCAST,NOARP,PROMISC,SLAVE,UP,LOWER_UP> mtu 9000 qdisc fq_codel master bond0 state UP group default qlen 1000
    link/ether 7e:9c:17:21:e2:27 brd ff:ff:ff:ff:ff:ff
    altname enp0s19
4: bond0: <BROADCAST,MULTICAST,PROMISC,MASTER,UP,LOWER_UP> mtu 9000 qdisc noqueue state UP group default qlen 1000
    link/ether 7e:9c:17:21:e2:27 brd ff:ff:ff:ff:ff:ff
6: sobridge: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
    link/ether 02:42:98:9a:2f:d7 brd ff:ff:ff:ff:ff:ff
    inet 172.17.1.1/24 brd 172.17.1.255 scope global sobridge
       valid_lft forever preferred_lft forever
7: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default
    link/ether 02:42:45:58:fc:84 brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/24 brd 172.17.0.255 scope global docker0
       valid_lft forever preferred_lft forever
```

cat /proc/net/bonding/bond0:
```
Ethernet Channel Bonding Driver: v5.15.0-103.114.4.el9uek.x86_64

Bonding Mode: load balancing (round-robin)
MII Status: up
MII Polling Interval (ms): 100
Up Delay (ms): 0
Down Delay (ms): 0
Peer Notification Delay (ms): 0

Slave Interface: ens19
MII Status: up
Speed: Unknown
Duplex: Unknown
Link Failure Count: 0
Permanent HW addr: 7e:9c:17:21:e2:27
Slave queue ID: 0
```
so-test:
```
Replaying PCAP(s) at 10 Mbps on interface bond0...
Actual: 111557 packets (12981286 bytes) sent in 10.38 seconds
Rated: 1249999.2 Bps, 9.99 Mbps, 10742.09 pps
Flows: 4102 flows, 394.99 fps, 2074477 flow packets, 45106 non-flow
Statistics for network device: bond0
        Successful packets:        55748
        Failed packets:            0
        Truncated packets:         0
        Retried packets (ENOBUFS): 0
        Retried packets (EAGAIN):  0
Replay completed. Warnings shown above are typically expected.
```

Some screenshots from influxdb:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/6805296/2010969b-1ee5-4b86-bd9b-7e1db387bf2f)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/6805296/6d4deb16-558d-447c-a0ae-51e8e7527ecc)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
Can you try rebooting?

**Reply by Mich-b** [reply]
Tried that twice, no improvement. 

Btw I think I had the same problem as https://github.com/Security-Onion-Solutions/securityonion/discussions/11109 but I simply ignored the error during installation.

**Reply by TOoSmOotH** [reply]
Are there logs in /nsm/zeek/logs/current? Is the eve log in /nsm/suricata being populated?

**Reply by Mich-b** [reply]
In /nsm/zeek/logs/current there is about 153Mb of logs, in /nsm/suricata there is 32Mb of logs. 
So they are indeed populated. How come I don't see them in the main dashboard or in the Kibana dashboards?

**Reply by dougburks** [reply, marked as answer]
Have you tried the hotfix?
https://blog.securityonion.net/2023/08/security-onion-2410-hotfix-20230821-now.html

If you continue to have problems after updating to the hotfix, here are some troubleshooting steps that might help:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts
